### ISSUE TITLE
Command templates have broken heading hierarchy

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-developer#394]
Single command: The ToC needs to be after the h1, and Global Parameters and Installing should be h2s.

![Screenshot 2023-11-21 at 5 27 43 PM](https://github.com/WordPress/wporg-developer/assets/1017872/2c48a5dd-f5f8-4a91-b77e-28b8adb56680)

Commands landing: 

![Image](https://github.com/WordPress/wporg-developer/assets/1017872/8c9a8d51-6dd4-485b-bc7e-a690a9b8f303)
